src/ILCompiler.ReadyToRun/src/Compiler/DependencyAnalysis/CoreCLRReadyToRunHeaderNode.cs

```diff
@@ -0,0 +1,119 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+
+using Internal.Runtime;
+using Internal.Text;
+using Internal.TypeSystem;
+
+namespace ILCompiler.DependencyAnalysis
+{
+    internal struct ReadyToRunHeaderConstants
+    {
+        public const uint Signature = 0x00525452; // 'RTR'
+
+        public const ushort CurrentMajorVersion = 2;
+        public const ushort CurrentMinorVersion = 1;
+    }
+
+    public class CoreCLRReadyToRunHeaderNode : ObjectNode, ISymbolDefinitionNode
+    {
+        struct HeaderItem
+        {
+            public HeaderItem(ReadyToRunSectionType id, ObjectNode node, ISymbolNode startSymbol)
+            {
+                Id = id;
+                Node = node;
+                StartSymbol = startSymbol;
+            }
+
+            readonly public ReadyToRunSectionType Id;
```

Comment:
Nit: we follow the [CoreFX coding style guidelines](https://github.com/dotnet/corefx/blob/bceb1d4d11b7f374498f15184499ff77f2c127ba/Documentation/coding-guidelines/coding-style.md). Visibility comes first and should be always specified (that's for the `_items` and `_target` below)

Reply:
Fixed